exp/querytenant/info.go

```diff
@@ -0,0 +1,90 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2024 Weaviate B.V. All rights reserved.
+//
+//  CONTACT: hello@weaviate.io
+//
+
+package querytenant
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+)
+
+const (
+	// v1/schema/collection/tenants
+	DefaultSchemaPath = "v1/schema/%s/tenants"
+)
+
+var ErrTenantNotFound = errors.New("tenant not found")
+
+type TenantInfo struct {
```

Comment:
Good point. Only problem is the weaviate go client already [depends on core weaviate as dependency](https://github.com/weaviate/weaviate-go-client/blob/8a4e2ba2477ce5b2c65861a389b5b9837305842a/go.mod#L12). It will be then cyclic dependencies.

Ideal way is to split the common code and put it in core weaviate, and use it both go-client and exp/querytenant packages I think.

Reply:
Got it, let's leave that for a future PR, perhaps leave a TODO or document it somewhere so we don't forget

Reply:
I'm fine with home baking our approach for now, but wouldn't the dep cycle only trigger if we import the exact same package twice ? As long as we're importing the client in the `exp` package it should be ok ?

Reply:
> but wouldn't the dep cycle only trigger if we import the exact same package twice ? As long as we're importing the client in the exp package it should be ok ?

That's true, package is the unit of dependency. But given we may use different types and helpers in our `exp` package going forward, almost always we end up in the state of cyclic dependency.

Ideally I want to expose this `schema` endpoint as grpc and use it in `querytenant`. I will add a TODO